RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -187,6 +187,7 @@ struct PurchaseInformation {
         case promotional
         case google
         case web
+        case webBilling
```

Comment:
I wonder if these 2 are different enough. Maybe something like: 
```suggestion
        case webExternal / externalWeb
        case rcBilling / rcWebBilling
```
What do you think?